pom.xml

```diff
@@ -138,6 +142,12 @@
         </dependency>
 
         <dependency>
+            <groupId>javax.mail</groupId>
```

Comment:
While sending response in GtfsFeed.java file from the function 

```@GET 
@Path("/{id : \\d+}/errorCount") 
@Produces(MediaType.APPLICATION_JSON)
public Response getFeedErrorCount(@PathParam("id") int id) {
         Session session = GTFSDB.InitSessionBeginTrans();
         GtfsFeedModel gtfsFeed = (GtfsFeedModel) session.createQuery(" FROM GtfsFeedModel
              WHERE feedId = " + id).uniqueResult();
         GTFSDB.commitAndCloseSession(session); 
         return Response.ok(gtfsFeed).build();
}
```

is throwing an exception, `javax.servlet.ServletException:org.glassfish.jersey.server.ContainerException:java.lang.NoClassDefFoundError: javax.mail.internet.MimeMultipart` if Java Mail API is not added.